src/headline_grabber/pipeline_steps/filter_max_results.py

```diff
@@ -0,0 +1,25 @@
+from src.headline_grabber.models.pipeline_context import PipelineContext
+from src.headline_grabber.pipeline_steps.pipeline_step import PipelineStep
+from src.headline_grabber.models.headline import Headline
+from typing import List
+
+class FilterMaxResults(PipelineStep):
+    def run(self, context: PipelineContext):
+        if context.user_input.entries is not None:
+            print(context)
+            subject_count = {}
+            fl: List[Headline] = []
```

Comment:
not sure what `fl` stands for, let's find a better naming for the variable, like `selected_headlines`, or something

Reply:
It stood for filter_list. The new name should represent that.